.github/workflows/kind.yaml

```diff
@@ -11,12 +11,16 @@ jobs:
     steps:
       - uses: actions/checkout@master
       - uses: engineerd/setup-kind@v0.5.0
+        with:
+          version: "v0.11.1"
       - name: Testing on KinD
         run: |
           export GO111MODULE=on
           export PATH=$PATH:$(go env GOPATH)/bin
           kubectl cluster-info
           kubectl get pods -n kube-system
+          kubectl wait --for=condition=Ready pods --all -n kube-system
+          kubectl get pods -n kube-system
```

Comment:
You should delete the get pods in the line 21, no?

Reply:
Maybe, I guess we can.

Reply:
Actually, maybe better to remove line 23. So we know, which pods wasn't ok, before.. ?